### PR TITLE
Edits to command help

### PR DESCRIPTION
**Pull request by bhardesty**
I did a comprehensive review of the rhoas CLI command help and made edits to make the help more consistent and to add some additional detail for some of the features.

I did not review the Service Registry commands, but those can be done in a future PR.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: pkg/localize/locales/en/cmd/acl.en.toml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 [kafka.acl.cmd]
 [kafka.acl.cmd.shortDescription]
-one = 'Kafka ACL management for users and service accounts'
+one = 'Manage Kafka ACLs for users and service accounts'
 
 [kafka.acl.cmd.longDescription]
 one = '''
-Set of commands that will let you manage Kafka ACLs.
-By default, every user and service account has limited access to their Kafka instance (Only describe permission is enabled for TOPIC, ACL, and GROUP).
+Manage Kafka Access Control Lists (ACLs). An ACL maps permitted operations on specified resources for a selected account or for all accounts in an organization.
```
Comment (wtrocki) [inline pkg/localize/locales/en/cmd/acl.en.toml:7]:
```suggestion
Manage Kafka Access Control Lists (ACLs). An ACL maps permitted operations on specified resources for a specified principal (usename or service account) or for all accounts in an organization.
```

File: pkg/localize/locales/en/cmd/acl.en.toml [hunk 2]
```diff
@@ -314,7 +315,7 @@ one = 'Deleting ACLs from Kafka instance "{{.Name}}"'
 one = 'Grant an account permissions to create and delete ACLs in the Kafka instance'
 
 [kafka.acl.grantAdmin.cmd.longDescription]
-one = 'This command grants a specified account permission to create and delete ACLs in a Kafka instance.'
+one = 'Grant admin rights to a user. Admin rights enable the user to create and delete Access Control Lists (ACLs) in a Kafka instance.'
```
Comment (wtrocki) [inline pkg/localize/locales/en/cmd/acl.en.toml:318]: Not 100% sure about "admin" rights
There are no typical "admin" rights in Kafka, but I think we mean here an "administrative" rights to cluster (AKA alter permission https://docs.confluent.io/platform/current/kafka/authorization.html). 

It think it is better to use full "administrative" rights as "admin" might be missleading for user thinking that they get admin rights (where all they get is to Cluster Alter right)

```suggestion
one = 'Grant administrative rights to a user. Admin rights enable the user to create and delete Access Control Lists (ACLs) in a Kafka instance.'
```

File: pkg/localize/locales/en/cmd/cluster.en.toml [hunk 3]
```diff
@@ -149,50 +142,38 @@ one = 'Connect your services to Kubernetes or OpenShift'
...
 
-  $ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all
+Finally, the RHOAS Operator creates a Kafka Request Object, which the Service Binding Operator (https://github.com/redhat-developer/service-binding-operator) uses to create a "ServiceBinding" object.
 
-Similarly for service registry you should execute the following command:
-
-  $ rhoas service-registry role add --role=DEVELOPER --service-account your-sa
+After running this command, you need to grant access for the service account that was created by the RHOAS Operator to access your application service:
 
+$ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all
```
Comment (wtrocki) [inline pkg/localize/locales/en/cmd/cluster.en.toml:157]: We are missing distiction between service registry and kafka access rights (This was specifically required depending on the type of the service we have.

Can we restore content starting from 

`NOTE: Created service account will need to be given permissions to specified service

**Review by wtrocki — APPROVED** [review]
Amazing job! Left very minor suggestions.

I would normally merge this but I think we need to keep sentence about service registry specific access rules for cluster connect.

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 4]
```diff
@@ -408,34 +412,32 @@ one = 'Name of the Kafka instance you want to view'
...
 one = '''
-List all Apache Kafka instances.
+List all Kafka instances.
 
-This command will provide a high level view of all Kafka instances.
-The fields displayed are: ID, Name, Owner, Status, Cloud Provider, Region.
-Use the describe command to view all fields for a specific instance.
+By default, this command lists the Kafka instances in a table, showing the ID, name, owner, status, cloud provider, and region.
 
-The instances are displayed by default in a table, but can also be displayed as JSON or YAML.
```
Comment (rkpattnaik780) [inline pkg/localize/locales/en/cmd/kafka.en.toml:429]: We should keep the text about JSON and YAML to be consistent with help text of other list commands.

File: pkg/localize/locales/en/cmd/cluster.en.toml (hunk 3 above)
Comment (bhardesty) [reply pkg/localize/locales/en/cmd/cluster.en.toml:157]: I think I captured the requested portion. Here's how the full long description now reads. Let me know if this isn't correct:

```
Connect your application services to an Kubernetes or OpenShift cluster.

Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/service-discovery#installing-the-rhoas-operator-on-openshift.

When you connect an application service to a Kubernetes or OpenShift cluster, this command uses the kubeconfig file (or the KUBECONFIG environment variable) to connect to the cluster and identify the context.

After identifying the context, the RHOAS Operator creates a service account and mounts it as a secret in your cluster. If your cluster has a service account already, the service account is refreshed.

Finally, the RHOAS Operator creates a Kafka Request Object, which the Service Binding Operator (https://github.com/redhat-developer/service-binding-operator) uses to create a "ServiceBinding" object.

After running this command, you need to grant access for the service account that was created by the RHOAS Operator to access your application service. For the Kafka application service, enter the following command:

  $ rhoas kafka acl grant-access --producer --consumer --service-account your-sa --topic all --group all

For the Service Registry application service, enter this command:

  $ rhoas service-registry role add --role=DEVELOPER --service-account your-sa
```

File: pkg/localize/locales/en/cmd/kafka.en.toml (hunk 4 above)
Comment (bhardesty) [reply pkg/localize/locales/en/cmd/kafka.en.toml:429]: I added this back in.

**Comment by bhardesty** [comment]
@wtrocki @rkpattnaik780 I made some updates based on your feedback - thanks! Let me know if everything looks good. Do I need to rebuild the help in order for the checks to pass? I was waiting until all of the reviews are complete.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**